scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,243 @@
+                # Python <3.11 fromisoformat needs "+00:00" not "Z"
+                if sys.version_info < (3, 11):
+                    dt_str = created_at_str.replace("Z", "+00:00")
+                else:
+                    dt_str = created_at_str
+                current_comment_dt = datetime.datetime.fromisoformat(dt_str)
+                if latest_created_at_obj is None or current_comment_dt > latest_created_at_obj:
+                    latest_created_at_obj = current_comment_dt
+            except ValueError:
+                sys.stderr.write(f"Warning: Could not parse timestamp: {created_at_str}\n")
+
+        # Get other comment details
+        user = comment.get("user", {}).get("login", "Unknown user")
+        path = comment.get("path", "N/A")
+        body = comment.get("body", "").strip()
+
+        if not body:
+            continue
+
+        diff_hunk = comment.get("diff_hunk")
+        html_url = comment.get("html_url", "N/A")
+        comment_id = comment.get("id")
+        in_reply_to_id = comment.get("in_reply_to_id")
+
+        print(f"## Comment by: **{user}** (ID: `{comment_id}`){f' (In Reply To: `{in_reply_to_id}`)' if in_reply_to_id else ''}\n")
+        if created_at_str:
+            print(f"*   **Timestamp**: `{created_at_str}`")
+        print(f"*   **Status**: `{status_text}`")
+        print(f"*   **File**: `{path}`")
+        print(f"*   **Line**: `{line_to_display}`") # Label changed from "Line in File Diff"
+        print(f"*   **URL**: <{html_url}>\n")
+
+        print("\n### Context:")
+        print("```") # Start of Markdown code block
+        if diff_hunk and diff_hunk.strip():
+            if args.context_lines == 0: # User wants the full hunk
+                print(diff_hunk)
+            else: # User wants N lines of context (args.context_lines > 0)
+                hunk_lines = diff_hunk.split('\n')
+                header_printed_this_time = False
+                if hunk_lines and hunk_lines[0].startswith("@@ "):
+                    print(hunk_lines[0])
+                    hunk_lines = hunk_lines[1:] # Operate on the rest of the hunk
+                    header_printed_this_time = True # Flag that header was output
+
+                if hunk_lines: # If there are lines left after potentially removing header
+                    lines_to_print_count = args.context_lines
+                    actual_trailing_lines = hunk_lines[-lines_to_print_count:]
+                    for line_content in actual_trailing_lines:
+                        print(line_content)
+                elif not header_printed_this_time :
+                    pass # Hunk was empty or only a header, already handled by outer 'else' or header print
+
+        else: # diff_hunk was None or empty
+            print("(No diff hunk available for this comment)")
+        print("```") # End of Markdown code block
+
+        print("\n### Comment:")
+        print(body)
+        print("\n---")
+
+    if latest_created_at_obj:
```

Comment:
Can we add a stderr output before this that tells you how many comments it exported? Because usually when I run this script I pipe the output to pbcopy (so I can paste it into my browser) so I only see the stderr, and it would be nice to see how many comments it retrieved.